Add tsprint v1.6.2

markthetech opened the pull request:
### Checklist

- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:

- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

I may need to change the cask name but I do not know how to.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

vitorgalvao commented on Casks/tsprint.rb:
```diff
@@ -0,0 +1,13 @@
+cask 'tsprint' do
+  version '1.6.2'
```
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

markthetech replied:
changed version to :latest and sha256 to :nocheck

vitorgalvao commented on Casks/tsprint.rb:
```diff
@@ -0,0 +1,13 @@
+
+  pkg 'TSPrintClient.pkg'
+
+  uninstall quit: 'com.terminalworks.TSPrintClient', trash: '/Applications/TSPrintClient.app'
```
Separate `trash` into another line.

markthetech replied:
I put trash onto a new line.

vitorgalvao commented on Casks/tsprint.rb:
```diff
@@ -9,5 +9,6 @@
 
-  uninstall quit: 'com.terminalworks.TSPrintClient', trash: '/Applications/TSPrintClient.app'
+  uninstall quit: 'com.terminalworks.TSPrintClient'
+  uninstall trash: '/Applications/TSPrintClient.app'
```
It should be just one `uninstall`. See other casks. What I meant was

```ruby
uninstall quit: 'com.terminalworks.TSPrintClient',
          trash: '/Applications/TSPrintClient.app'
```

reitermarkus replied:
Why isn't his using `uninstall pkgutil:` when a `pkg` installer was used?

vitorgalvao replied:
@reitermarkus Good point. I didn’t even read `quit` correctly, just outright assumed it was `pkgutil` (too many reviews of similar things, I guess).

markthetech replied:
I'm not using pkgutil: because in my testing it didn't successfully uninstall it.

vitorgalvao replied:
@markthetech Even if it did not uninstall everything (not unusual), it’s highly unlikely the only thing the `pkg` does is put `.app` in `Applications`. It’s very likely it’s installing other things, and `pkgutil` might take care of those.

markthetech replied:
Alright that makes sense. I'll add it.

vitorgalvao commented:
You need to rerun the commands every time you update the cask. There are now errors that can be fixed by `brew cask style --fix`.

markthetech commented:
@vitorgalvao I'll do the style fix. I didn't understand how so thanks for letting me know. This is my first time so I'm a bit new at this.

markthetech commented:
Fixed the style problems. Added pkgutil as well. I left trash because without it it will not successfully delete the .app from the applications folder.
